src/test/cli/help-long-expected

```diff
@@ -129,6 +129,9 @@ Experimental (Unstable and may change or be removed at any time, regardless of S
       --strace-logging-mode <mode>
           Log the syscalls for each process to individual "strace" files [default: "off"]
 
+      --stub-syscalls <syscalls>
+          List of syscall numbers to add "return 0" stubs for. [default: []]
```

Comment:
I don't see any existing syscall stubs in Shadow that we'd want to use this for. But we might want to add another config option in the future for adding stubs for socket options, and in that case we probably would want to add defaults for for those socket options.

> they would probably get confused and accidentally drop the defaults by writing `--stub-syscalls=666` instead of `--stub-syscalls=x,y,z,666`

Yeah, this is similar to the `--args` option of `tornettools simulate`, where adding new args drops the default args. I'm not sure what the solution to this is.